pkgs/applications/editors/emacs/elisp-packages/manual-packages/lua/default.nix

```diff
@@ -24,7 +25,10 @@ melpaBuild {
 
   nativeBuildInputs = [ pkg-config ];
 
-  buildInputs = [ lua ];
+  buildInputs = [
+    emacs
+    lua
+  ];
```

Comment:
Should we add `emacs` to `buildInputs` by default since it is needed if an elisp package builds a dynamic module.

Reply:
I'm also kind of surprised these modules explicitly need to link to emacs, since it will already have the symbols it needs loaded when it gets dlopen'd from emacs.